### DISCUSSION TITLE
AUTOSAVE BUG

### DISCUSSION THREAD

**Discussion by caoguojun0** [zed-industries/zed#12053] [Feature Requests]
Autosaving with a time delay  makes the cursor immediately teleport to the start of the line, as if the file gets reloaded, this is really annoying.
i got `remove_trailing_whitespace_on_save` on true
How i can avoid this issue?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
